hermes-upload.ps1

```diff
@@ -267,12 +338,12 @@ function delete-if-exists
 }
 $scriptPath = split-path -parent $MyInvocation.MyCommand.Definition
 
-$username = "admin"
-$password = "admin"
-$server = "ftp://ftp.votinginfoproject.org/"
+$userId = "<INSERT USER-ID HERE>"
+$apiKey = "<INSERT API-KEY HERE>"
+$server = "https://staging-upload.votinginfoproject.org/upload"
 
 $fips = "12345"
-$electionDate = "2016-11-08"
+$electionDate = "2019-11-08"
```

Comment:
This answers my previous question, but 💥 the question suddenly expands into: I expect somebody to remove this assignment, so if that happens, is not having the `$electionDate` going to cause a bad request to be made?